system-tests/project-fixtures/angular/src/app/svg.component.ts

```diff
@@ -0,0 +1,7 @@
+import { Component } from "@angular/core";
```

Comment:
Can delete `svg.component.ts`, this what just a test to verify that svg templates load. It's a react svg anyway 😆